sdk/core/core-amqp/test/errors.spec.ts

```diff
@@ -30,6 +31,32 @@ describe("Errors", function() {
       translatedError.message.should.equal(ehError.message);
     });
 
+    it("Sets retryable to true, if input is custom error and name is OperationTimeoutError", function() {
+      const err = new Error("error message");
+      err.name = "OperationTimeoutError";
+      const translatedError = Errors.translate(err);
+      translatedError.message.should.equal(err.message);
+      translatedError.stack!.should.equal(err.stack);
+      translatedError.retryable.should.equal(true);
+    });
+
+    it("Sets retryable to true, if input is custom error and name is InsufficientCreditError", function() {
+      const err = new Error("error message");
+      err.name = "InsufficientCreditError";
+      const translatedError = Errors.translate(err);
+      translatedError.message.should.equal(err.message);
+      translatedError.stack!.should.equal(err.stack);
+      translatedError.retryable.should.equal(true);
+    });
+
+    it("Sets retryable to false, if input is the custom AbortError", function() {
+      const err = new AbortError("error message");
+      const translatedError = Errors.translate(err);
+      translatedError.message.should.equal(err.message);
+      translatedError.stack!.should.equal(err.stack);
+      translatedError.retryable.should.equal(false);
+    });
```

Comment:
Since the above 3 are custom errors, also add a check that ensures that the name on the translated error is the same as the name on the original error